solutions/security/get-started/enable-threat-intelligence-integrations.md

```diff
@@ -19,4 +19,78 @@ $$$agent-ti-integration$$$
+
+:::{image} ../../../images/serverless--getting-started-threat-intelligence-view.png
+:alt: The Threat Intelligence view on the Overview dashboard
+:class: screenshot
+:::
+
+There are a few scenarios when data won’t display in the Threat Intelligence view:
+
+* If you’ve chosen a time range that doesn’t contain threat indicator event data, you are prompted to choose a different range. Use the date and time picker in the {{security-app}} to select a new range to analyze.
+* If the {{agent}} or {{filebeat}} agent hasn’t ingested Threat Intel module data yet, the threat indicator event counts won’t load. You can wait for data to be ingested or reach out to your administrator for help resolving this.
+
+
+## Add an {{agent}} integration [agent-ti-integration]
+
+1. Install a [{{fleet}}-managed {{agent}}](https://www.elastic.co/guide/en/fleet/current/install-fleet-managed-elastic-agent.html) on the hosts you want to monitor.
+2. In the Threat Intelligence view, click **Enable sources** to view the Integrations page. Scroll down and select **Elastic Agent only** to filter by {{agent}} integrations.
+
+    ::::{tip}
+    If you know the name of {{agent}} integration you want to install, you can search for it directly. Alternatively, choose the **Threat Intelligence** category to display a list of available [threat intelligence integrations](https://docs.elastic.co/en/integrations/threat-intelligence-intro).
+
+    ::::
+
+3. Select an {{agent}} integration, then complete the installation steps.
+4. Return to the Threat Intelligence view on the Overview dashboard. If indicator data isn’t displaying, refresh the page or refer to these [troubleshooting steps](../../../troubleshoot/security/indicators-of-compromise.md#troubleshoot-indicators-page).
+
+
+## Add a {{filebeat}} Threat Intel module integration [ti-mod-integration]
+
+% Substeps in step 2 will require inline versioning. Remember to update them when we have more guidance on handling line-level differences.
+
+1. Set up the [{{filebeat}} agent](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation-configuration.html) and enable the Threat Intel module.
+
+    ::::{note}
+    For more information about enabling available threat intelligence filesets, refer to [Threat Intel module](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html).
+
+    ::::
+
+2. Update the `securitySolution:defaultThreatIndex` [advanced setting](../../../solutions/security/get-started/configure-advanced-settings.md#update-threat-intel-indices) by adding the appropriate index pattern name after the default {{fleet}} threat intelligence index pattern (`logs-ti*`):
+
+    * If you’re *only* using {{filebeat}} version 8.x, add the appropriate {{filebeat}} threat intelligence index pattern. For example, `logs-ti*`, `filebeat-8*`. 
```

Comment:
(Left a comment [above](https://github.com/elastic/docs-content/pull/405/files#diff-1c25c0a522389515072904057037f8d4bc2a951b4fb850fa55718396f1eb5fd2R64), but leaving a comment here as well for extra visiblity)

This might require versioning annotation since it applies to ECH and users who are upgrading to 9.0 might still use 8.x versions of Filebeat.